docker-for-mac/install.md

```diff
@@ -76,6 +76,41 @@ The Docker Desktop installation includes
 
 Congratulations! You are now successfully running Docker Desktop.
 
+## Switch between Stable and Edge versions
+
+Docker Desktop allows you to switch between Stable and Edge releases. However, **you can only have one version of Docker Desktop installed at a time**. Switching between Stable and Edge versions can destabilize your development environment, particularly in cases where you switch from a newer (Edge) channel to older (Stable).
+
```

Comment:
s/channel to older/channel to an older channel

Reply:
Agree. Will update accordingly.

Reply:
Fixed